### PR TITLE
duckdb: add page

### PR DESCRIPTION
**Pull request by szarnyasg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.8.1

### CODE REVIEW

File: pages/common/duckdb.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`duckdb`
+
+- Start an interactive shell on a database file. If the file does not exist, a new database is created:
+
+`duckdb {{path/to/dbfile.[db|duckdb]}}`
+
+- Load CSV or Parquet file to a table and quit:
+
+`duckdb {{path/to/dbfile.[db|duckdb]}} "{{CREATE OR REPLACE TABLE tbl AS FROM 'data_source.[csv|parquet]'}}"`
```
Comment (Mause) [inline pages/common/duckdb.md:16]: Missing the command flag?

**Review by Mause — CHANGES_REQUESTED** [review]

File: pages/common/duckdb.md (hunk 1 above)
Comment (szarnyasg) [reply pages/common/duckdb.md:16]: Passing a command without `-cmd` / `-c` / `-s`: 

```console
$ duckdb dbfile.duckdb "SELECT 42 AS x";
┌───────┐
│   x   │
│ int32 │
├───────┤
│    42 │
└───────┘
```

Comment (szarnyasg) [reply pages/common/duckdb.md]: That said, the positional version can be confusing as `duckdb "SELECT 42 AS x"` will create a database file named `SELECT 42 AS x`. So I added the `-c` switch before the query.

**Review by Mause — APPROVED** [review]

File: pages/common/duckdb.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Run SQL queries in file on database and keep interactive shell open:
+
+`duckdb {{path/to/dbfile.[db|duckdb]}} -init {{path/to/script.sql}}`
+
+- Open database in read-only mode:
+
+`duckdb -readonly {{path/to/dbfile.[db|duckdb]}}`
+
+- Show help:
```
Comment (kbdharun) [inline pages/common/duckdb.md:30]:
```suggestion
- Display help:
```
Minor suggestion, we use the generic phrasing `Display help` over `Show help` across our pages.

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

File: pages/common/duckdb.md (hunk 2 above)
Comment (szarnyasg) [reply pages/common/duckdb.md]: Thanks! Good point, I commited it.

**Review by Mause — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**